java/test/jmri/jmrit/logixng/actions/ActionLightTest.java

```diff
@@ -41,7 +41,7 @@ public void testLightState() {
         
         boolean hasThrown = false;
         try {
-            ActionLight.LightState.get(Light.UNKNOWN);
+            ActionLight.LightState.get(100);
```

Comment:
should this is a symbolic constant?  Or get a comment about `100` being the right number?

Reply:
ActionLight has an enum LightState that corresponds to the known states of Light. This part of the test tests that `LightState.get(state)` throws an exception if the state is unknown. Before, I used the constant `Light.UNKNOWN`, but since @jerryg2003 wants to be able to set that state, I cannot use it anymore. So I now use a number that's unknown to LightState.

I have added a comment about this explaining the number 100.